pkg/distributor/forwarding/forwarding.go

```diff
@@ -223,6 +224,8 @@ func (f *forwarder) Forward(ctx context.Context, endpoint string, rules validati
 		return in, errCh
 	}
 
+	spanlog, ctx := spanlogger.NewWithLogger(ctx, f.log, "forwarder.Forward")
```

Comment:
nit
```suggestion
	spanlog, ctx := spanlogger.NewWithLogger(ctx, f.log, "forwarder.Forward()")
```

Reply:
we actually don't use this style elsewhere. I will remove it from `request.do` span too.